### ISSUE TITLE
[Bug] Version 0.5.1 has incomplete metric information.

### ISSUE THREAD

**Issue by tensorflowt** [sgl-project/sglang#13485]
### Checklist

- [x] I searched related issues but found no solution.
- [x] The bug persists in the latest version.
- [x] Issues without environment info and a minimal reproducible demo are hard to resolve and may receive no feedback.
- [x] If this is not a bug report but a general question, please start a discussion at https://github.com/sgl-project/sglang/discussions. Otherwise, it will be closed.
- [x] Please use English. Otherwise, it will be closed.

### Describe the bug

```
curl http://localhost:30000/metrics
# HELP sglang:num_running_reqs The number of running requests.
# TYPE sglang:num_running_reqs gauge
sglang:num_running_reqs{engine_type="unified",model_name="Qwen-32B",pp_rank="0",tp_rank="0"} 1.0
# HELP sglang:num_used_tokens The number of used tokens.
# TYPE sglang:num_used_tokens gauge
sglang:num_used_tokens{engine_type="unified",model_name="Qwen-32B",pp_rank="0",tp_rank="0"} 1.0
# HELP sglang:token_usage The token usage.
# TYPE sglang:token_usage gauge
sglang:token_usage{engine_type="unified",model_name="Qwen-32B",pp_rank="0",tp_rank="0"} 0.0
# HELP sglang:gen_throughput The generation throughput (token/s).
# TYPE sglang:gen_throughput gauge
sglang:gen_throughput{engine_type="unified",model_name="Qwen-32B",pp_rank="0",tp_rank="0"} 0.8371284457403783
# HELP sglang:num_queue_reqs The number of requests in the waiting queue.
# TYPE sglang:num_queue_reqs gauge
sglang:num_queue_reqs{engine_type="unified",model_name="Qwen-32B",pp_rank="0",tp_rank="0"} 0.0
# HELP sglang:num_grammar_queue_reqs The number of requests in the grammar waiting queue.
# TYPE sglang:num_grammar_queue_reqs gauge
sglang:num_grammar_queue_reqs{engine_type="unified",model_name="Qwen-32B",pp_rank="0",tp_rank="0"} 0.0
# HELP sglang:cache_hit_rate The prefix cache hit rate.
# TYPE sglang:cache_hit_rate gauge
sglang:cache_hit_rate{engine_type="unified",model_name="Qwen-32B",pp_rank="0",tp_rank="0"} 0.0
# HELP sglang:spec_accept_length The average acceptance length of speculative decoding.
# TYPE sglang:spec_accept_length gauge
sglang:spec_accept_length{engine_type="unified",model_name="Qwen-32B",pp_rank="0",tp_rank="0"} 0.0
# HELP sglang:total_retracted_reqs The total number of retracted requests due to kvcache full.
# TYPE sglang:total_retracted_reqs gauge
sglang:total_retracted_reqs{engine_type="unified",model_name="Qwen-32B",pp_rank="0",tp_rank="0"} 0.0
# HELP sglang:num_prefill_prealloc_queue_reqs The number of requests in the prefill prealloc queue.
# TYPE sglang:num_prefill_prealloc_queue_reqs gauge
sglang:num_prefill_prealloc_queue_reqs{engine_type="unified",model_name="Qwen-32B",pp_rank="0",tp_rank="0"} 0.0
# HELP sglang:num_prefill_infight_queue_reqs The number of requests in the prefill infight queue.
# TYPE sglang:num_prefill_infight_queue_reqs gauge
sglang:num_prefill_infight_queue_reqs{engine_type="unified",model_name="Qwen-32B",pp_rank="0",tp_rank="0"} 0.0
# HELP sglang:num_decode_prealloc_queue_reqs The number of requests in the decode prealloc queue.
# TYPE sglang:num_decode_prealloc_queue_reqs gauge
sglang:num_decode_prealloc_queue_reqs{engine_type="unified",model_name="Qwen-32B",pp_rank="0",tp_rank="0"} 0.0
# HELP sglang:num_decode_transfer_queue_reqs The number of requests in the decode transfer queue.
# TYPE sglang:num_decode_transfer_queue_reqs gauge
sglang:num_decode_transfer_queue_reqs{engine_type="unified",model_name="Qwen-32B",pp_rank="0",tp_rank="0"} 0.0
```

This section lacks metrics such as end-to-end latency, TTFT latency, and number of tokens output per second.

### Reproduction

```
CUDA_VISIBLE_DEVICES=0,1 \
nohup \
python3 -m sglang.launch_server \
    --model-path /mnt/data/models/Qwen/Qwen3-30B-A3B-Instruct-2507-FP8 \
    --mem-fraction-static 0.8 \
    --enable-torch-compile \
    --tp 2 \
    --port 30000 \
    --host 0.0.0.0 \
    --log-level info \
    --enable-metrics \
    --enable-p2p-check \
    > /tmp/s.logs 2>&1 &
```

### Environment

/opt/venv/lib/python3.12/site-packages/torch/cuda/__init__.py:63: FutureWarning: The pynvml package is deprecated. Please install nvidia-ml-py instead. If you did not install pynvml directly, please report this to the maintainers of the package that installed pynvml for you.
  import pynvml  # type: ignore[import]
Python: 3.12.9 (main, Mar 17 2025, 21:01:58) [Clang 20.1.0 ]
CUDA available: True
GPU 0,1: NVIDIA GeForce RTX 4090
GPU 0,1 Compute Capability: 8.9
CUDA_HOME: /usr/local/cuda
NVCC: Cuda compilation tools, release 12.4, V12.4.131
CUDA Driver Version: 550.90.07
PyTorch: 2.8.0+cu128
sglang: 0.5.1
sgl_kernel: 0.3.5
flashinfer_python: 0.2.11.post3
triton: 3.4.0
transformers: 4.55.2
torchao: 0.9.0
numpy: 1.26.4
aiohttp: 3.11.14
fastapi: 0.115.11
hf_transfer: 0.1.9
huggingface_hub: 0.35.1
interegular: 0.3.3
modelscope: 1.24.0
orjson: 3.11.4
outlines: 0.1.11
packaging: 24.2
psutil: 7.0.0
pydantic: 2.11.9
python-multipart: 0.0.20
pyzmq: 26.3.0
uvicorn: 0.34.0
uvloop: 0.21.0
vllm: 0.10.1+cu124
xgrammar: 0.1.23
openai: 1.99.1
tiktoken: 0.9.0
anthropic: 0.72.1
litellm: Module Not Found
decord: 0.6.0
NVIDIA Topology:
	GPU0	GPU1	CPU Affinity	NUMA Affinity	GPU NUMA ID
GPU0	 X 	PHB	0-63	0		N/A
GPU1	PHB	 X 	0-63	0		N/A

Legend:

  X    = Self
  SYS  = Connection traversing PCIe as well as the SMP interconnect between NUMA nodes (e.g., QPI/UPI)
  NODE = Connection traversing PCIe as well as the interconnect between PCIe Host Bridges within a NUMA node
  PHB  = Connection traversing PCIe as well as a PCIe Host Bridge (typically the CPU)
  PXB  = Connection traversing multiple PCIe bridges (without traversing the PCIe Host Bridge)
  PIX  = Connection traversing at most a single PCIe bridge
  NV#  = Connection traversing a bonded set of # NVLinks

Hypervisor vendor: KVM
ulimit soft: 1048576

**Comment by frankie-ys** [comment]
I meet same question like this in Version 0.5.5 post3. I request to localhost:8080/metrics, it responses 404 not found.
